packages/docs-infra/src/pipeline/transformTypescriptToJavascript/removeTypes.ts

```diff
@@ -103,13 +103,13 @@ export async function removeTypes(
   const standardPrettierOptions: PrettierOptions = {
     parser: 'babel',
     singleQuote: true,
-    plugins: [prettierPluginEstree, parserBabel],
+    plugins: [pluginBabel, pluginEstree],
   };
 
   // If `prettierConfig` is *explicitly* true (as opposed to truthy), it means the user has opted in
```

Comment:
Can't we just resolve the config of the project? Prettier has [API](https://prettier.io/docs/api#prettierresolveconfigfileurlorpath--options) for that.